src/components/Search/Form.js

```diff
@@ -0,0 +1,63 @@
+import { render } from '../../shared/dom';
+import { redirect } from '../../';
+import { routes } from '../../route';
+import ConsoleLogger from '../../utils/ConsoleLogger';
+
+const IDLE_TIMEOUT_LIMIT_MS = 800;
+const IDLE_TIMEOUT_CHECK_MS = 50;
+
+export const SearchForm = ({ renderOn, searchInputValue }) => {
+    const template = `
+        <div class="search-form" id="search-form">
+            <input
+                type="text"
+                id="search-keywords"
+                placeholder="WYSZUKAJ"
+                value="${searchInputValue}"
+                autocomplete="off"
+            >
+
+            <button>
```

Comment:
Also I guess that should be clickable so I think you could leave a TODO comment in here,
Also I think that you could use the Button component here, pass the svg and the text as the text argument and style it via passed arguments